view/frontend/templates/recaptcha.phtml

```diff
@@ -0,0 +1,13 @@
+<?php // phpcs:disable
+
+/** @var $viewModel \CardknoxDevelopment\Cardknox\ViewModel\Data */
+$viewModel = $block->getViewModel();
+$isEnabledReCaptcha = $viewModel->isEnabledReCaptcha();
+$selectReCaptchaSource = $viewModel->getSelectReCaptchaSource();
+
+
+if ($isEnabledReCaptcha && $selectReCaptchaSource == "recaptcha.net") : ?>
+    <script>
+        require(['https://www.recaptcha.net/recaptcha/api.js']);

```

Comment:
Is this the same file as the js below?

Reply:
@mstein-ck, I updated the code and removed both recaptcha__en.js & api.js